flink-next-gen-connector-influxdb/src/main/java/org/apache/flink/streaming/connectors/influxdb/sink/writer/InfluxDBWriter.java

```diff
@@ -69,7 +69,7 @@ public void write(final IN in, final Context context) throws IOException {
 
     @Override
     public List<IN> snapshotState() throws IOException {
```

Comment:
I guess this should be 
```suggestion
    public List<Point> snapshotState() throws IOException {
```
though, check if possible with interface